src/Glpi/Asset/Capacity/IsReservableCapacity.php

```diff
@@ -52,6 +53,12 @@ public function getIcon(): string
         return Reservation::getIcon();
     }
 
+    #[Override]
+    public function getDescription(): string
+    {
+        return __("End users can reserve this asset");
```

Comment:
```suggestion
        return __("These assets can be made reservable");
```
Maybe need to clarify that this allows assets of this type to be optionally reservable, rather than making all of them reservable just by enabling this.